src/commands/run.js

```diff
@@ -25,20 +25,6 @@ export async function run(yargs) {
     cmd += ' debug';
   }
 
-  const devSettings = 'settings/dev.settings.json';
-  const prodSettings = 'settings/settings.json';
-
-  if (args.settings) {
-    Log.info(`\nUsing settings file at ${Log.magenta(args.settings)}\n`);
-    cmd += ` --settings ${args.settings}`;
-  } else if (exists(prodSettings)) {
```

Comment:
We still want this first block so that `--settings` works.

```js
  if (args.settings) {
    Log.info(`\nUsing settings file at ${Log.magenta(args.settings)}\n`);
    cmd += ` --settings ${args.settings}`;
  }
```